src/web/lib/components/App/index.js

```diff
@@ -73,9 +74,29 @@ export const AppComponent = ({
       )}
     <AppBackground {...{ theme }} />
     {!hasExtension && <Banner {...{ addonUrl, bottom: false }} />}
-    <div className="app-content">
+    {loaderDelayExpired &&
+    <div className="app__content">
       <AppHeader {...{ hasExtension }} />
       <BrowserPreview {...{ theme, setSelectedColor, size: 'large' }}>
+        <div className="app__theme-element-pickers">
+          <ThemeColorsEditor
+            {...{
+              theme,
+              selectedColor,
+              setColor,
+              setSelectedColor,
+              undo,
```

Comment:
These undo/redo props don't need to be passed into `ThemeColorsEditor` since the buttons got pulled into their own component and moved into `App`